sdk/examples/wallet/offline_signing/1_prepare_transaction.rs

```diff
@@ -30,7 +31,7 @@ async fn main() -> Result<()> {
 
     let outputs = vec![SendAmountParams::new(
         // Address to which we want to send the amount.
-        "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu".to_string(),
+        Bech32Address::try_from_str("rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu").unwrap(),
```

Comment:
Can't we use this here?
```suggestion
        Bech32Address::try_from_str("rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu")?,
```